recipe/meta.yaml

```diff
@@ -22,16 +22,14 @@ requirements:
   run:
     - python >=3.7
     - django >=3.2
-    - beautifulsoup4 >=4.12.2
-    - pillow >=9.5.0
```

Comment:
It looks to me that these are still listed as dependencies in version 23.3 but not in 23.4: https://github.com/zostera/django-bootstrap5/blob/v23.4/pyproject.toml#L29 Can you update this to be for version 23.4?

Reply:
Hi @swainn 
It's only in hatch env not in dependencies - 
https://github.com/zostera/django-bootstrap5/blob/v23.3/pyproject.toml#L28
I think this fix is valid for both 23.3 and 23.4 
I will create another PR for 23.4 too - but I believe this can be merged too. 
Could you take a another look and see if I am reading this right -

Reply:
Ah, I see it now. Good catch. Happy to have you help as a maintainer too, btw.